app/src/main/java/com/example/notebook/view/fragments/my_notes/creation_note/CreationNoteFragment.kt

```diff
@@ -46,6 +43,6 @@ class AddingNoteFragment : BaseFragment<FragmentAddNoteBinding>(FragmentAddNoteB
         }
     }
 
-    private fun isNoteNotEmpty(): Boolean =
-        binding.editTitle.text?.isNotEmpty() == true && binding.editNote.text?.isNotEmpty() == true
+    private fun isNoteEmpty(): Boolean =
+        binding.editTitle.text.isNullOrEmpty() && binding.editNote.text.isNullOrEmpty()
```

Comment:
I think your inversion is kind of invalid here because to check if the note is not empty you needed both fields to be filled but now to check if a note is empty you also check if both fields are empty. I mean, at least one text field is not empty then the program would regard your note as not being empty which I consider to be at least unexpected behaviour